### PR TITLE
*: enable prevote by default

### PR DESCRIPTION
**Pull request by nolouch**
for minimizing disruptions.

### CODE REVIEW

File: server/config.go [hunk 1]
```diff
@@ -93,6 +93,11 @@ type Config struct {
 	TickInterval typeutil.Duration `toml:"tick-interval"`
 	// ElectionInterval is the interval for etcd Raft election.
 	ElectionInterval typeutil.Duration `toml:"election-interval"`
+	// DisablePrevote is true to disable Raft Pre-Vote.
+	// If enabled, Raft runs an additional election phase
+	// to check whether it would get enough votes to win
+	// an election, thus minimizing disruptions.
+	DisablePreVote bool `json:"disable-pre-vote"`
```
Comment (siddontang) [inline server/config.go]: why not using PreVote directly?

Comment (nolouch) [reply server/config.go]: I want to enable it by default. use `PreVote` should distinguish whether the key exists in the TOML actually is false or not set it.

Comment (siddontang) [reply server/config.go]: so, if not, use true directly.

We must explicitly set to false if we want to disable it. 

Of course, you can add prevote=true in the example configuration.

File: conf/config.toml [hunk 2]
```diff
@@ -19,6 +19,8 @@ tso-save-interval = "3s"
 
 namespace-classifier = "table"
 
+pre-vote = true
```
Comment (disksing) [inline conf/config.toml]: How about 'enable-prevote'?

**Comment by disksing** [comment]
LGTM.

**Comment by nolouch** [comment]
PTAL @siddontang

**Pull request merged**